napari/_qt/widgets/qt_progress_bar.py

```diff
@@ -13,9 +13,12 @@
 class ProgressBar(QWidget):
     """QProgressBar with QLabels for description and ETA."""
 
-    def __init__(self, parent=None) -> None:
+    def __init__(self, parent=None, prog=None) -> None:
         super().__init__(parent)
         self.setAttribute(QtCore.Qt.WA_DeleteOnClose)
+
+        self.prog = prog
+
         self.pbar = QProgressBar()
```

Comment:
```suggestion
        self.qt_progress_bar = QProgressBar()
```
I don't feel so strongly about this one, but could be considered - could also be left